src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -3,6 +3,7 @@ import styles from './SkillStyle';
 import ISO6391 from 'iso-639-1';
 import SelectField from 'material-ui/SelectField';
 import { Paper } from 'material-ui';
+// eslint-disable-next-line
```

Comment:
I see that eslint has been disabled for several imports. Can you please tell what issues you were having? Thanks :-)

Reply:
Throws unused warning in the console since `Card` is not really directly rendered in the DOM but saved in the state instead.